core/src/dird/ua.h

```diff
@@ -280,6 +282,8 @@ class RunContext {
   bool nextpool_set = false;
   bool accurate_set = false;
   bool ignoreduplicatecheck_set = false;
+  bool allow_mixed_priority = false;
+  bool allow_mixed_priority_set = false;
```

Comment:
Yes, good point.
However, it just occurred to me that the allowmixedpriority UA run command parameter (and therefore this code too) isn't even needed because I can just grab it later on from the rc.consolidate_job resource which simplifies things.

Reply:
These two bools have now been removed.